docs/admin-guide/loqusdb_integration.md

```diff
@@ -5,7 +5,19 @@
 ## Setup
 
 To avoid dependency conflicts loqusdb should be installed in an environment separate from scout.
-Install loqudb according to the instructions on the package homepage. This can be done using the same environment or another virtual environment, also the same mongodb process can be used for the loqus database however if large datasets(1000s of whole genomes) are expected it is preferable to keep these separate. In the config `config.py` give the connection information like:
+Install loqudb according to the instructions on the package homepage. This can be done using the same environment or another virtual environment, also the same mongodb process can be used for the loqus database however if large datasets(1000s of whole genomes) are expected it is preferable to keep these separate. 
```

Comment:
```suggestion
Install loqudb according to the instructions on the package homepage. This can be done using the same environment or another virtual environment, also the same mongodb process can be used for the loqus database however if large datasets(1000s of whole genomes) are expected it is preferable to keep loqusdb and scout in separate instances. 
```